dfgviewer/Configuration/TypoScript/Page/header.ts

```diff
@@ -20,7 +20,7 @@ page {
 	}
 	headerData {
 		10 = TEXT
-		10.value = <link rel="shortcut icon" type="image/png" href="EXT:dfgviewer/Resources/Public/Images/dfgviewerFavicon.png">
+		10.value = <link rel="shortcut icon" type="image/png" href="/typo3conf/ext/dfgviewer/Resources/Public/Images/dfgviewerFavicon.png">
```

Comment:
That's `page.shortcutIcon` is intended to:

`page.shortcutIcon = EXT:dfgviewer/Resources/Public/Images/dfgviewerFavicon.png`